ene_017_rw1_energy_facility_emissions/README.md

```diff
@@ -0,0 +1,17 @@
+This file describes the data pre-processing that was done to [Energy Facility Emissions (US)](https://ghgdata.epa.gov/ghgp/main.do#) for [display on Resource Watch](https://bit.ly/3kNAD3i).
+
+The source provided this dataset as an xls file accessed through its [website](https://ghgdata.epa.gov/ghgp/main.do#). To access the file select the "Export Data" option and then select "All Reporting Years". 
+
+Below, we describe the main actions performed to process the xls file:
+
+1. Convert the years in the 'Year' column to datetime objects and store them in a new column 'datetime'.
```

Comment:
Is there an advantage to using datetimes for years instead of just having them as integers? I typically think it makes the date harder to format in the widget editor (because really you would only want to show the year and not the rest of the date) and maybe be confusing for someone if they thought the data was only associated with a particular day instead of the whole year.